doc/release-checklist.md

```diff
@@ -45,13 +45,15 @@ See this page for a good overview: https://deps.rs/repo/github/sharkdp/bat
 
 ## Release
 
-- [ ] Create a tag and push it to the remote `git tag vX.Y.Z; git push --tags`.
+- [ ] Create a tag and push it to the remote `git tag vX.Y.Z; git push origin vX.Y.Z`.
```

Comment:
Maybe this is bad practice, but I do sometimes tend to name branches after upcoming releases, i.e. with the same name as the tag. In that case, I would need to `git push origin tag vX.Y.Z`.

Reply:
I would agree it is bad practice :)

But doesn't hurt to change the command to what works in even more cases, so I'll do that.